packages/bot-skeleton/src/services/tradeEngine/utils/helpers.js

```diff
@@ -90,15 +88,20 @@ const getBackoffDelay = (error, delayIndex) => {
 
     const maxExpTries = 4;
     const exponentialIncrease = 2 ** delayIndex + offset;
+    const log_type = 'passed_error';
+    const extra = {};
+    let msg;
 
     if (errorCode === 'RateLimit' || delayIndex < maxExpTries) {
-        notifyRetry(localize('Rate limit reached for'), error, exponentialIncrease);
+        msg = 'Rate limit reached for';
+        extra.longcode = `${msg}: ${error.error.msg_type}, retrying in ${exponentialIncrease} s`;
```

Comment:
Could we merge `msg` and `extra.longcode` and localize the string? e.g.

```suggestion
extra.longcode = localize(`Rate limit reached for: {{ error_type }}, retrying in {{ timeout }}s`, {
    error_type: error.error.msg_type,
    timeout: exponentialIncrease,
});
```